qa/tasks/mgr/test_module_selftest.py

```diff
@@ -52,12 +52,6 @@ def test_influx(self):
 
     def test_diskprediction_local(self):
         self._load_module("selftest")
-        python_version = self.mgr_cluster.mon_manager.raw_cluster_cmd(
-            "mgr", "self-test", "python-version")
-        if tuple(int(v) for v in python_version.split('.')) >= (3, 8):
```

Comment:
if the solution is to *skip* 3.8, probably instead of removing the whole `if` block, we should s/>=/==/.